### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bdsomer**
The current `README.md` file shows the use of Node.js-style callbacks in the `Jimp.read` function. Sadly, this is simply incorrect as of January 10th, 2018 in the current `master` branch (see that there is no callback argument [here](https://github.com/oliver-moran/jimp/blob/master/index.js#L325)).

An alternative solution would be to implement callbacks into the code, but I believe that moving towards promises will increase the longevity of this package and encourage other package maintainers/collaborators to do the same.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,19 +49,23 @@ Jimp.read("./path/to/image.jpg").then(function (image) {
...
+}).catch(function (err) {
+    // handle an exception
 });
 ```
 
 
 ### Basic methods ###
 
-Once the callback is filed or the promise fulfilled, the following methods can be called on the image:
+Once the the promise is fulfilled, the following methods can be called on the image:
```
Comment (edi9999) [inline README.md]: typo here : the the => the

Comment (edi9999) [reply README.md]: ping

**Comment by bdsomer** [comment]
@edi9999 updated. sorry for the late response.

**Comment by edi9999** [comment]
Thanks !

**Pull request merged**